vgen_process_simulink_model.m

```diff
@@ -123,8 +123,13 @@
 project_revision = mp.modelName + "_" + target;
 sopcinfo_file = hdlpath + "/quartus/" + target + '_system.sopcinfo';
 disp("Generating device tree source file")
-disp(python + mp.dtogen_path + filesep + "generate.py -s " + sopcinfo_file + " -r " + project_revision + " -o " + hdlpath)
-system(python + mp.dtogen_path + filesep + "generate.py -s " + sopcinfo_file + " -r " + project_revision + " -o " + hdlpath);
+if target == "reflex" || target == "audioblade"
+    extra_flag = " -d " + project_revision
+else
+    extra_flag = " -r " + project_revision;
+end
```

Comment:
Am I correct that, for the audiomini, the rbf name argument is used to determine the dts name, while for the arria10 devices, the `-d` flag is used?

If so, it seems like it would be cleaner to always use `-d` for the dts name, then optionally include the rbf argument when devices support it.

Reply:
> Am I correct that, for the audiomini, the rbf name argument is used to determine the dts name, while for the arria10 devices, the -d flag is used?    

The `-d` flag can always set dts name, but is optional if the `-r` flag is used to specify an rbf file with the given name will be used.
> If so, it seems like it would be cleaner to always use -d for the dts name, then optionally include the rbf argument when devices support it.

 If someone was using `dtogen` directly they can already do that.

Reply:
Gotcha. I would argue for not being able to set the dts name with `-r`, and making `-d` required.